encoder/vaapiencoder_base.cpp

```diff
@@ -417,6 +427,46 @@ Encode_Status VaapiEncoderBase::getOutput(VideoEncOutputBuffer * outBuffer, bool
     return ENCODE_SUCCESS;
 }
 
+#else
+Encode_Status VaapiEncoderBase::getOutput(VideoEncOutputBuffer * outBuffer, VideoEncMVBuffer * MVBuffer, bool withWait)
+{
+    unsigned char *data = NULL;
```

Comment:
is any possible to use getOutput(VideoEncOutputBuffer * outBuffer, bool withWait) ? or we can move common code to other function.